operatorhub-resources/wso2am-operator/1.0.1/wso2am-operator.v1.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,333 @@
+                - create
+                - update
+                - delete
+                - patch
+                - watch
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - namespaces
+                - secrets
+                - events
+                - services
+                - serviceaccounts
+                - configmaps
+                - persistentvolumeclaims
+              verbs:
+                - get
+                - list
+                - create
+                - update
+                - delete
+                - patch
+                - watch
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - extensions
+              resources:
+                - ingresses
+              verbs:
+                - get
+                - list
+                - create
+                - update
+                - delete
+                - patch
+                - watch
+            - apiGroups:
+                - apiextensions.k8s.io
+              resources:
+                - customresourcedefinitions
+              verbs:
+                - get
+                - list
+                - create
+                - update
+                - delete
+                - patch
+                - watch
+            - apiGroups:
+                - apim.wso2.com
+              resources:
+                - apimanagers
+                - apimanagers/finalizers
+              verbs:
+                - get
+                - list
+                - create
+                - update
+                - delete
+                - patch
+                - watch
+      deployments:
+        - name: wso2am-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                app: wso2am-controller
+            template:
+              metadata: 
+                labels:
+                  app: wso2am-controller
+              spec:
+                containers:
+                  - name: wso2am-controller
+                    image: 'wso2/k8s-wso2am-operator:1.0.1'
+                    imagePullPolicy: Always
+                    env:
+                      - name: WATCH_NAMESPACE
+                        value: ''
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: wso2am-operator
+                serviceAccountName: wso2am-pattern-1-svc-account
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: true
+    - type: AllNamespaces
+      supported: true
```

Comment:
Shall we have a new line here, if these are not auto-generated files? check other places as well.

Reply:
Fixed with https://github.com/wso2/k8s-wso2am-operator/pull/81